extensions/book/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "book",
```

Comment:
>book [](start = 13, length = 4)

This seems a little generic - it should generally be something that gives a general idea of what it does. So like notebook-tree-view would be better